subsys/bluetooth/host/direction.c

```diff
@@ -189,6 +200,151 @@ static int hci_df_set_adv_cte_tx_enable(struct bt_le_ext_adv *adv,
 				   buf, NULL);
 }
 
+#if defined(CONFIG_BT_DF_CONNECTIONLESS_CTE_RX)
+static int hci_df_set_cl_cte_rx_enable(struct bt_le_per_adv_sync *sync, bool enable,
+				       const struct bt_df_per_adv_sync_cte_rx_param *params)
+{
+	struct bt_hci_cp_le_set_cl_cte_sampling_enable *cp;
+	struct bt_hci_cmd_state_set state;
+	const uint8_t *ant_ids;
+	struct net_buf *buf;
+	int err;
+
+	if (enable) {
+		if (!(params->cte_type &
+		      (BT_DF_CTE_RX_TYPE_AOA | BT_DF_CTE_RX_TYPE_AOD_1US |
+		       BT_DF_CTE_RX_TYPE_AOD_2US))) {
+			return -EINVAL;
+		}
+
+		if (params->cte_type & BT_DF_CTE_RX_TYPE_AOA) {
+			if (df_ant_info.num_ant <
+				    DF_SAMPLING_ANTENNA_NUMBER_MIN ||
+			    !BT_FEAT_LE_ANT_SWITCH_RX_AOA(bt_dev.le.features)) {
```

Comment:
Minor:
```suggestion
			if (df_ant_info.num_ant < DF_SAMPLING_ANTENNA_NUMBER_MIN ||
			    !BT_FEAT_LE_ANT_SWITCH_RX_AOA(bt_dev.le.features)) {
```
To make it significantly more readable without crossing the 100 char rule.

Reply:
Updates, unfortunately .clang-format has 80 characters limit still. That is the cause for strange formatting in the PR.